packages/typo3-docs-theme/resources/template/structure/navigation/meta-menu.html.twig

```diff
@@ -0,0 +1,28 @@
+
+{%- set copySources = getSettings('copy_sources') -%}
+{%- set howtoEditButton = getSettings('how_to_edit') -%}
+{%- set gitHubLink = getEditOnGitHubLink() -%}
+
+{%- if (copySources or howtoEditButton or gitHubLink)  %}
+<div aria-label="Meta navigation" class="main_menu  d-block d-lg-none" role="navigation">
+    <hr/>
+    <ul class="menu-level-1">
+        <p class="caption">Contributors Corner</p>
```

Comment:
Is a <p> inside <ul> allowed per HTML spec? I thought a <li> would be mandatory; maybe better place it outside?

Reply:
ah, right

Reply:
Done